nepal_compliance/nepali_date_utils/nepali_date.py

```
@@ -6,17 +6,33 @@
+        """Coerce a value to date via Frappe."""
+        return frappe.utils.getdate(v)
+    def _to_datetime(v):
+        """Coerce a value to datetime via Frappe."""
+        return frappe.utils.get_datetime(v)
+    def _throw(msg):
+        """Raise a Frappe validation error."""
+        frappe.throw(msg)
+    def _log(msg):
+        """Write a Frappe error log entry."""
+        frappe.log_error(msg)
 except Exception:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)